src/commands/tenmans.ts

```diff
@@ -186,7 +186,7 @@ class VoteQueueButtonAction extends VoteQueueAction<ButtonInteraction> {
       });
 
       await queueChannel.send({
-        content: `<@${roleId}> that Radianite must be ours! A queue has been created!`,
+        content: `<@${tenmansRole.name}> that Radianite must be ours! A queue has been created!`,
```

Comment:
Change to this syntax: https://stackoverflow.com/questions/58749366/how-do-i-mention-a-role-with-discord-js